libs/sdk-ui-charts/src/highcharts/adapter/plugins/group-categories-wrapper.ts

```diff
@@ -20,42 +18,4 @@ export function groupCategoriesWrapper(Highcharts: any): void {
 
         hideGridInXAxis.call(this);
     });
-
-    wrap(Highcharts.Tick.prototype, "getPosition", function (
```

Comment:
This was currently causing a bug. Previously it was probably fixing some issue that has been resolved in Highcharts already.